README.md

```diff
@@ -1 +1,319 @@
 # AbsaOSS/k3d-action
+Easy to use GitHub action allows you to test single/multi-cluster apps with additional private registry support.
```

Comment:
imho this action has many possible applications, and we should not limit it to testing scope only.
```suggestion
A GitHub Action to run lightweight ephemeral Kubernetes clusters during workflow
```